e2e/security/security.go

```diff
@@ -3,14 +3,12 @@
 // LICENSE.md file distributed with the sources of this project regarding your
 // rights to use or distribute this software.
 
-// +build seccomp
+// TODO: --> +build seccomp
```

Comment:
what's the "TODO"?

Reply:
This file originally had a comment:

```go
// +build seccomp
```

CCI did not like that, and when I removed that line, Circle CI was happy. So dose that comment do anything important?